content/en/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -0,0 +1,53 @@
+---
+reviewers:
+- maplain
+title: Service Internal Traffic Policy
+content_type: concept
+weight: 10
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Service Internal Traffic Policy_ enables cluster internal traffic routing to node
+local endpoints only. This approach tries to keep traffic closer to where it
+originated from. This can help to reduce costs and improve performance.
+
+<!-- body -->
+
+## Motivation
+
+Kubernetes Service is usually backed up by endpoints scattered around the
+whole cluster.
+_Service Internal Traffic Policy_ provides a mechanism to help keep cluster
+internal traffic within the same node it originated from. The "internal" traffic
+here refers to traffic originated from Pods in the current cluster.
+
+## Using Service Internal Traffic Policy
+
+You can enable Internal Traffic Policy for a Service by setting the
+`spec.internalTrafficPolicy` to `Local`. This tells kube-proxy to only use node
+local endpoints for cluster internal traffic. Importantly, this Service will
+appear to have zero endpoint for any Pod accessing this Service on a node, when
+this Service doesn't have any local endpoint on the node, but does have endpoints
+on other nodes.
+
+## How it Works
+
+kube-proxy filters the endpoints it routes to based on the
+`spec.internalTrafficPolicy` setting. When it's "Local", only node local
+endpoints are considered.
```

Comment:
What happens when it's not set to `Local`?

Reply:
updated